Add type definition about workflow app.

yokotaso opened the pull request.

yokotaso committed:
Add type definition about workflow app.

teppeis commented on tsconfig.json:
```diff
@@ -57,6 +57,11 @@
+  "files" : [
+    "src/schedule.d.ts",
+    "src/workflow.d.ts"
+  ],
```
@yokotaso VS Code and users of this package needs `/// <reference />`, this config file is only for internal developers of this package.

yokotaso committed:
include workflow.d.ts, schedule.d.ts with /// <reference path=.... />

youtr-s commented on src/workflow.d.ts:
```diff
@@ -0,0 +1,132 @@
+
+            interface RouteNaviValueItem {
+                name: string;
+                type: 'DATETIME';
```
type の値は'DATETIME' ではなく 'ROUTE_NAVI'かと思います。

youtr-s commented on src/workflow.d.ts:
```diff
@@ -0,0 +1,132 @@
+                    | 'APPROVE'
+                    | 'REJECT'
+                    | 'WITHDRAW'
+                    | 'CANCEL'[];
```
listで選択できる値に'CONFIRM','ACKNOWLEDGE'がありません。
https://developer.cybozu.io/hc/ja/articles/115001457503#step4

yokotaso committed:
Fix bug

- type of RouteNaviValueItem is wrong
- list of AvailableOperations is missing some type
